.github/workflows/integration_omnibus.yml

```diff
@@ -457,7 +457,7 @@ jobs:
       - uses: ./.github/actions/codebuild-docker-run
         name: Run Container
         with:
-          image: ${{ steps.login-ecr.outputs.registry }}/aws-lc/ubuntu:22.04
+          image: ${{ steps.login-ecr.outputs.registry }}/aws-lc/ubuntu-24.04_integration
```

Comment:
If I'm not mistaken, we migrated Docker image build to GHA. Shouldn't we make use of that?